src/main/resources/application.yml

```diff
@@ -35,23 +39,24 @@ chain:
   gasPriceMultiplier: ${IEXEC_GAS_PRICE_MULTIPLIER:1.0} # txs will be send with networkGasPrice*gasPriceMultiplier, 4.0 means super fast
   gasPriceCap: ${IEXEC_GAS_PRICE_CAP:22000000000} #in Wei, will be used for txs if networkGasPrice*gasPriceMultiplier > gasPriceCap
 
-workers:
-  askForReplicatePeriod: ${IEXEC_ASK_REPLICATE_PERIOD:5000}
-  requiredWorkerVersion: ${IEXEC_CORE_REQUIRED_WORKER_VERSION:} #leave empty will allow any worker version
-
 resultRepository:
   protocol: ${IEXEC_RESULT_REPOSITORY_PROTOCOL:http}
   host: ${IEXEC_RESULT_REPOSITORY_HOST:localhost}
   port: ${IEXEC_RESULT_REPOSITORY_PORT:18090}
 
+ipfs:
+  host: ${IEXEC_IPFS_HOST:127.0.0.1}
+  port: ${IEXEC_IPFS_PORT:5001}
+
 sms:
   protocol: ${IEXEC_SMS_PROTOCOL:http}
   host: ${IEXEC_SMS_HOST:localhost}
   port: ${IEXEC_SMS_PORT:5000}
 
-ipfs:
-  host: ${IEXEC_IPFS_HOST:127.0.0.1}
-  port: ${IEXEC_IPFS_PORT:5001}
+scone:
```

Comment:
Isn't it
cas:
  scone:
?

Reply:
No CAS is a scone service. So CAS under scone (we have also LAS under scone in the worker side)